### DISCUSSION TITLE
Unable to run localAI through obsidian

### DISCUSSION THREAD

**Discussion by falapark** [logancyang/obsidian-copilot#170] [Q&A]
Hi,
I've been trying to run copilot plugin. I am using Windows 10 and installed LocalAi through wsl2 with docker. When I test that the modle was installed correctly through WSL2 terminal it works, I can chat with the model but when it comes to Obsidian and the plugin, it just wont connect. I entered the correct model and the correct proxy. Any type of help or idea on how I could possibly get this to work would be appreciated. 

![1](https://github.com/logancyang/obsidian-copilot/assets/59677336/b63642bc-d6d4-4614-a5eb-606fc3e89665)
![1](https://github.com/logancyang/obsidian-copilot/assets/59677336/1f6f80a0-0c72-420c-bc37-8f30087ac7fd)
![1](https://github.com/logancyang/obsidian-copilot/assets/59677336/4f2354c5-6236-4a23-9cb8-43ebbb2af753)

**Comment by logancyang** [comment]
Could be a known issue where you need to put something in the OpenAI API key field in the settings, even something random should do.
